content/slides/data-inspection.qmd

```diff
@@ -18,35 +18,383 @@ the same datum using e.g. `gdalwarp`, `ogr2ogr`.
+  bed#source=Mass conservation (Mathieu Morlighem)
+  bed#standard_name=bedrock_altitude
+  bed#units=meters
+  bed#_FillValue=-9999
+  mapping#false_easting=0
+  mapping#false_northing=0
+  mapping#geoid=eigen-6c4
+  mapping#grid_mapping_name=polar_stereographic
+  mapping#inverse_flattening=298.2794050428205
+  mapping#latitude_of_projection_origin=90
+  mapping#semi_major_axis=6378273
+  mapping#standard_parallel=70
+  mapping#straight_vertical_longitude_from_pole=-45
+  NC_GLOBAL#Author=Mathieu Morlighem
+  NC_GLOBAL#Conventions=CF-1.7
+  NC_GLOBAL#Data_citation=Morlighem M. et al., (2017), BedMachine v3: Complete bed topography and ocean bathymetry mapping of Greenland from multi-beam echo sounding combined with mass conservation, Geophys. Res. Lett., 44, doi:10.1002/2017GL074954. (http://onlinelibrary.wiley.com/doi/10.1002/2017GL074954/full)
+  NC_GLOBAL#ice_density (kg m-3)=917
+  NC_GLOBAL#license=No restrictions on access or use
+  NC_GLOBAL#no_data=-9999
+  NC_GLOBAL#nx=10218
+  NC_GLOBAL#ny=18346
+  NC_GLOBAL#proj4=+init=epsg:3413
+  NC_GLOBAL#Projection=Polar Stereographic North (70N, 45W)
+  NC_GLOBAL#sea_water_density (kg m-3)=1023
+  NC_GLOBAL#spacing=150
+  NC_GLOBAL#Title=BedMachine Greenland
+  NC_GLOBAL#version=20-Apr-2021 (v4.10)
+  NC_GLOBAL#xmin=-652925
+  NC_GLOBAL#ymax=-632675
+  x#long_name=Cartesian x-coordinate
+  x#standard_name=projection_x_coordinate
+  x#units=meter
+  y#long_name=Cartesian y-coordinate
+  y#standard_name=projection_y_coordinate
+  y#units=meter
+Image Structure Metadata:
+  COMPRESSION=DEFLATE
+  INTERLEAVE=BAND
+Corner Coordinates:
+Upper Left  ( -997978.940, -329743.071) (116d42'57.18"W, 80d19'10.86"N)
+Lower Left  ( -997978.940,-3611073.668) ( 60d26'56.41"W, 56d22'33.59"N)
+Upper Right ( 1228573.966, -329743.071) ( 29d58'34.22"E, 78d17'47.47"N)
+Lower Right ( 1228573.966,-3611073.668) ( 26d12'37.67"W, 55d48' 2.10"N)
+Center      (  115297.513,-1970408.369) ( 41d39' 4.26"W, 71d55'29.45"N)
+Band 1 Block=256x256 Type=Float32, ColorInterp=Gray
+  Min=-5571.673 Max=3672.355
+  Minimum=-5571.673, Maximum=3672.355, Mean=-307.314, StdDev=1161.000
+  NoData Value=-9999
+  Overviews: 7422x10938, 3711x5469, 1856x2735, 928x1368
+  Unit Type: meters
+  Metadata:
+    grid_mapping=mapping
+    long_name=bed topography
+    NETCDF_VARNAME=bed
+    source=Mass conservation (Mathieu Morlighem)
+    standard_name=bedrock_altitude
+    STATISTICS_MAXIMUM=3672.35546875
+    STATISTICS_MEAN=-307.31420082049
+    STATISTICS_MINIMUM=-5571.6728515625
+    STATISTICS_STDDEV=1160.9999064556
+    STATISTICS_VALID_PERCENT=57.73
+    units=meters
+    _FillValue=-9999
+```
 
-...
+::: {.notes}
+* Grid size, in "pixels"
+* Well-Known Text represents CRS with hierarchical data, which `gdalinfo`
+nicely indents for us. The "top-level" EPSG is shown as 3413.
+* Pixel size, in projected units
+* Global metadata: _TODO: Mention of the difference namespaces, specifically
+	call out the `grid_mapping` namespace's significance?_
```

Comment:
These metadata come from cf conventions followed in the source data for this geotiff (a netcdf).

Reply:
We chatted on zoom and we need to determine whether this is a "gotcha" worth mentioning in the slides/speaker notes. Also, is it something we can easily fix in our QGreenland processing pipelines?